src/explorer/testExplorer.ts

```diff
@@ -100,15 +100,9 @@ export class TestExplorer implements TreeDataProvider<ITestItem>, Disposable {
                 if (result) {
                     switch (result.status) {
                         case TestStatus.Pass:
-                            return {
-                                dark: this._context.asAbsolutePath(path.join('resources', 'media', 'dark', 'pass.svg')),
-                                light: this._context.asAbsolutePath(path.join('resources', 'media', 'light', 'pass.svg')),
-                            };
+                            return new ThemeIcon('pass', new ThemeColor('charts.green'));
                         case TestStatus.Fail:
-                            return {
-                                dark: this._context.asAbsolutePath(path.join('resources', 'media', 'dark', 'error.svg')),
-                                light: this._context.asAbsolutePath(path.join('resources', 'media', 'light', 'error.svg')),
-                            };
+                            return new ThemeIcon('pass', new ThemeColor('charts.red'));
```

Comment:
Is this `'pass'` intentional vs `'error'`?

Reply:
Good catch!